src/main/kotlin/com/avail/compiler/AvailCompiler.kt

```diff
@@ -3564,6 +3564,12 @@ class AvailCompiler(
 			onSuccess(solutions) { this.rollbackModuleTransaction(it) }
 		}
 		recordExpectationsRelativeTo(start)
+		if (loader.lexicalScanner().allVisibleLexers.isEmpty())
+		{
+			start.expected(
+				STRONG,
+				"module to export at least 1 lexer in order to handle command")
+		}
```

Comment:
I would like Mark to review this, to make sure that it's in the right place doing the right thing at the right time. The compiler is really tricky. :)

Reply:
Quick note that this inclusion was accidental -- I have another branch for experimenting with workbench tweaks -- I will probably roll it back and wait until I have a PR for that branch since I haven't even analyzed it myself.

Reply:
Okay, good idea. Roll it back and we'll come back to it when you're ready with your workbench updates. Separately, if you are interested in working on workbench items, maybe you'd like to join us for the Anvil IDE project? It is coming along nicely, and could really benefit from your Avail interest and web expertise. Contact me privately if you're interested. :)